paper.md

```diff
@@ -0,0 +1,159 @@
+    affiliation: "1, 3"
+  - name: Carl Christofferson
+    orcid: 0009-0009-1164-3494
+    affiliation: "1, 2"
+  - name: Andrew Connolly
+    orcid: 0000-0001-5576-8189
+    affiliation: "1, 2"
+  - name: Jeremy Kubica
+    orcid: 0009-0009-2281-7031
+    affiliation: "1, 3"
+  - name: Olivia Lynn
+    orcid: 0000-0001-5028-146X
+    affiliation: "1, 3"
+  - name: Konstantin Malanchev
+    orcid: 0000-0001-7179-7406
+    affiliation: "1, 3"
+  - name: Alex Malz
+    orcid: 0000-0002-8676-1622
+    affiliation: "1, 3"
+  - name: Rachel Mandelbaum
+    orcid: 0000-0003-2271-1527
+    affiliation: "1, 3"
+  - name: Sean McGuire
+    orcid: 0009-0005-8764-2608
+    affiliation: "1, 3"
+  - name: Chris Wenneman
+    orcid: 0009-0002-6218-5007
+    affiliation: "1, 2"
+affiliations:
+  - name: LINCC-Frameworks
+    index: 1
+  - name: DiRAC Institute and the Department of Astronomy, University of Washington, 3910 15th Ave NE, Seattle, WA 98195, USA
+    index: 2
+  - name: The McWilliams Center for Cosmology & Astrophysics, Department of Physics, Carnegie Mellon University, Pittsburgh, PA 15213, USA
+    index: 3
+date: 28 February 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+The creation of healthy code is vital for its successful long term use in scientific research. To maximize impact throughout the community, software packages must be accurate, usable, and maintainable. Here we discuss several engineering processes that are important for developing healthy software. Unfortunately these processes often require configuration leading to short-term overhead for new projects. We introduce the LINCC-Frameworks Python Project Template, a configurable template designed for scientific software projects that greatly simplifies adopting such practices by automating the setup and configuration of important code health tools.
+
+# Statement of need
+
+Software has long played a vital role in analyzing scientific data and driving new discoveries. As the scientific community continues to build upon established projects and create novel algorithms, there is a need to ensure the accuracy, repeatability, usability, and maintainability of the software. The first factor is required for the validity of the scientific results, while the latter factors enable the software to have a broad, sustained impact.
+
+The LINCC-Frameworks Python Project Template (LF-PPT) was originally created with the needs of astronomers in mind, however, it became apparent that it is broadly applicable for many scientific use cases. The LF-PPT codifies our best engineering practices because we were unable to find existing tooling that met our needs. Many other templates exist for specific applications that include non-trivial amounts of code, but we wanted a template that 1) is agnostic to specific applications 2) includes tooling needed for healthy software 3) doesn’t preclude use of other application-specific templates and 4) is updatable as best practices and tooling evolve. 
```

Comment:
```suggestion
The LINCC-Frameworks Python Project Template (LF-PPT) was originally created with the needs of astronomers in mind; however, it became apparent that it is broadly applicable for many scientific use cases. The LF-PPT codifies our best engineering practices because we were unable to find existing tooling that met our needs. Many other templates exist for specific applications that include non-trivial amounts of code, but we wanted a template that 1) is agnostic to specific applications, 2) includes tooling needed for healthy software, 3) does not preclude use of other application-specific templates, and 4) is updatable as best practices and tooling evolve. 
```

Reply:
(just punctuation and "doesn't" -> "does not" for journal level of formality)